.circleci/config.yml

```diff
@@ -34,8 +23,37 @@ jobs:
           paths:
             - node_modules
             - app/node_modules
-          key: v1-dependencies-{{ checksum "yarn.lock" }}-{{ checksum "app/yarn.lock"}}
+            - ../.cache/Cypress
+          key: dependencies-{{ checksum "yarn.lock" }}-{{ checksum "app/yarn.lock"}}
 
       # run tests!
       - run: yarn eslint
       - run: yarn test-chrome-headless
+      - run:
+          name: Build the app so that server can be started
+          command: cd app && yarn build && cd ../
+      - run:
+          name: Start server with correct env for cypress tests
+          command: >
+            yarn generate-cert && ADALITE_ENABLE_HTTPS=true
```

Comment:
I'd extract this into a shell script and reuse it in package.json, so it isn't duplicated and CI and local cypress:run remains aligned

Reply:
It irked me to put cypress:dev here, since circleCI doesn't need dev environment, webpack watching, etc. I tried out some scripts, but it's not very cross-platform friendly, what would you propose? Stick to unix's "... | xargs"?